upgrade/Upgrade-6.0.1.php

```
@@ -53,5 +62,60 @@ function upgrade_module_6_0_1(Mollie $module): bool
         }
     }
 
+    $sql = '
+    SELECT COUNT(*) > 0 AS count
+    FROM information_schema.columns
+    WHERE TABLE_SCHEMA = "' . _DB_NAME_ . '" AND table_name = "' . _DB_PREFIX_ . 'mol_order_payment_fee";
+    ';
+
+    /** only add it if it doesn't exist */
+    if (!(int) Db::getInstance()->getValue($sql)) {
+        $sql = 'ALTER TABLE ' . _DB_PREFIX_ . 'mol_order_fee MODIFY id_mol_order_fee INT(64)';
+
+        try {
+            if (!Db::getInstance()->execute($sql)) {
+                return false;
+            }
+        } catch (Exception $e) {
+            return false;
+        }
+
+        $sql = 'ALTER TABLE ' . _DB_PREFIX_ . 'mol_order_fee DROP PRIMARY KEY';
+
+        try {
+            if (!Db::getInstance()->execute($sql)) {
+                return false;
+            }
+        } catch (Exception $e) {
+            return false;
+        }
+
+        $sql = '
+        ALTER TABLE ' . _DB_PREFIX_ . 'mol_order_fee
+        CHANGE order_fee fee_tax_incl decimal(20,6)  NOT NULL,
+        CHANGE id_mol_order_fee id_mol_order_payment_fee INT AUTO_INCREMENT PRIMARY KEY,
+        ADD COLUMN id_order INT(64) NOT NULL,
+        ADD COLUMN fee_tax_excl decimal(20,6) NOT NULL;
+        ';
+
+        try {
+            if (!Db::getInstance()->execute($sql)) {
+                return false;
+            }
+        } catch (Exception $e) {
+            return false;
+        }
+
+        $sql = 'RENAME TABLE ' . _DB_PREFIX_ . 'mol_order_fee TO ' . _DB_PREFIX_ . 'mol_order_payment_fee';
+
+        try {
+            if (!Db::getInstance()->execute($sql)) {
+                return false;
+            }
+        } catch (Exception $e) {
+            return false;
+        }
```

silent ignoring exceptions is very bad practice, you should atleast use logger to log an error what happened

---

Added message logging, didn't think of this. :D